package.json

```diff
@@ -18,6 +18,7 @@
     "@deck.gl/core": "8.1.0",
     "@deck.gl/layers": "8.1.0",
     "@hubmap/vitessce-image-viewer": "^0.2.1",
+    "@loadable/component": "^5.12.0",
```

Comment:
Why is react.lazy not sufficient?

[React](https://reactjs.org/docs/code-splitting.html#reactlazy) says:
> If you want to do code-splitting in a server rendered app, we recommend Loadable Components.

and we're not server rendered.

Reply:
I did try to use React.lazy and Suspense first but all the variations I tried produced errors:

![Screen Shot 2020-05-01 at 12 00 55 PM](https://user-images.githubusercontent.com/7525285/80819815-b7ee4f80-8ba3-11ea-9857-68ceeb34a4ad.png)

![Screen Shot 2020-05-01 at 12 00 40 PM](https://user-images.githubusercontent.com/7525285/80819827-be7cc700-8ba3-11ea-939a-25e740c64856.png)


![Screen Shot 2020-05-01 at 12 01 43 PM](https://user-images.githubusercontent.com/7525285/80819783-a86f0680-8ba3-11ea-888e-240a9f818ef9.png)

I think it may have something to do with the fact that the HiGlassComponent is not the "default export" of the higlass package? Maybe you can identify something I am doing wrong in the code in these three screenshots.

Reply:
Nothing off the top of my head. Just filed #527, and maybe we'll get back to this at some point. Thanks for looking at it. 

--> Could you add a comment in the code pointing to the issue? Thanks!

Reply:
Thanks for prompting me to investigate this a bit further. I just closed that issue and made this commit https://github.com/hubmapconsortium/vitessce/pull/523/commits/f368c54f4fb9c794307858f179bbf349b39aa54b which should resolve this. HiGlass just needs a "proxy" export where it can be the default export.

Reply:
I also verified in the developer tools that there is still a separate chunk of JS loaded, and it is the same 7.8 MB as before.